packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -253,7 +249,7 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
         <div className="flex flex-col">
           <NoteBox>
             After approval, you&apos;ll see a second prompt in your wallet for
-            the {isDepositMode ? 'deposit' : 'withdrawal'} transaction.
+            the {transferMode} transaction.
```

Comment:
Users will see "the teleport transaction" here. We've been calling it LayerLeap in the UI, we should be consistent with this.